content/guides/ethereum/tutorials/track_trx.md

```diff
@@ -130,7 +120,7 @@ Use the dfuse client with GraphQL query and set our transaction hash as a variab
                 hash:  transactionHash
             }
         });
-        await stream.join();
+        await stream.join();  // continues until disconnect and complete
```

Comment:
Well, the stream join is not related to disconnect, so I would probably remove that.

And in this case, the stream never completes since it runs forever.

Reply:
Gotcha, in this case, I simply copied the comment that was in the original `src/App.js` file. Should I correct that one as well in this case?

Reply:
I think it was changed for the title to respect casing of title.

Reply:
Oups, commented on wrong box! Both should be changed.

```suggestion
        await stream.join();  // awaits stream completion, which is never in this particular case
```